apps/web/tests/dashboards/dashboard-create.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { expect, test } from '@playwright/test';
+
+test.describe('Create Dashboard', () => {
+    test('새 대시보드 생성하기 - Entire Workspace / Single Project 대시보드', async ({ page }) => {
+        page.goto('/');
+
+        // Login
+        await test.step('enter user information', async () => {
+            await page.getByPlaceholder('User ID').click();
+            await page.getByPlaceholder('User ID').fill(process.env.USERNAME as string);
+            await page.getByPlaceholder('User ID').press('Tab');
+            await page.getByPlaceholder('Password').fill(process.env.PASSWORD as string);
+
+            // click the sign in button
+            const navigationPromise = page.waitForNavigation();
+            await page.getByRole('button', { name: 'Sign in', exact: true }).click();
+            await navigationPromise;
+
+            // go to the next page
+            await expect(page).toHaveURL(/home-dashboard/);
+        });
+
+        // Create Dashboard - Scope
+        await test.step('1.상단 GNB > Dashboards > Create New Dashboard 페이지 이동', async () => {
```

Comment:
Question: When writing the test description, I think we need to see if Korean is available!

Reply:
It was written in Korean to sync with the test scenario, but I think I should check it out!